db/migrate/20161119134300_add_organizer_to_workshops.rb

```diff
@@ -0,0 +1,5 @@
+class AddOrganizerToWorkshops < ActiveRecord::Migration
+  def change
+    add_column :workshops, :organizer, :string
```

Comment:
Does it really break stuff? This migration comes before 20161123124500_change_workshop_to_event. At that point in migrations, there is no table "events". The column is added to workshops. Afterwards, the table workshops is renamed to events.